packages/terra-card/src/CardPadding.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
```

Comment:
We still need it for when different child elements have different padding applied to them like how it is done in terra-site/examples/card/CardPaddingHR